stdlib/universe/universe.flux

```diff
@@ -237,6 +237,80 @@ builtin string : (v: A) => string
 builtin time : (v: A) => time
 builtin uint : (v: A) => uint
 
+// display converts any value into a string representation.
+// This function differs from the `string` function in that it will convert any value
+// to a string recusively for all composite types.
+//
+//  Basic types are converted to a string directly.
+//  Bytes are represented as a string of lowercase hexadecimal characters prefixed with `0x`.
+//  Composite types (i.e. arrays, dictionaries, records) will be represented in a syntax similar
+//  to the equivalent Flux literal representation.
+//
+//  It is not expected that the representation can be parsed back into the original value.
+//  The representation may span multiple lines.
+//  The representation is not stable across Flux versions.
```

Comment:
```suggestion
// display returns the Flux literal representation of any value as a string.
//
// Basic types are converted directly to a string.
// Bytes types are represented as a string of lowercase hexadecimal characters prefixed with `0x`.
// Composite types (arrays, dictionaries, and records) are represented in a syntax similar
// to their equivalent Flux literal representation.
//
// Note the following about the resulting string representation:
// - It cannot always be parsed back into the original value.
// - It may span multiple lines.
// - It may change between Flux versions.
//
// `display()` differs from `string()` in that `string()` recursively converts values inside
// composite types to strings.
```